tasks/internal/request-and-upload-signature/request-and-upload-signature.yaml

```diff
@@ -79,8 +85,14 @@ spec:
       description: The file where the signing response should be placed
       default: "signing_response.json"
       type: string
+    - name: pubtools-sign-config-path
```

Comment:
What is this for? It's not listed in the README. Is it just meant as a sort of constant that is used in the steps so they use the same place? Maybe a steptemplate env var would be more suitable for this?

https://tekton.dev/docs/pipelines/tasks/#specifying-a-step-template

Reply:
Hmm, ok, so you have both an empty dir mounted to /shared and a shared data workspace. The config in the task is hardcoded to `/shared/pubtools-sign-config.yaml`:
```
--config-file "/shared/pubtools-sign-config.yaml"
```

So I would say, if you have a parameter for this, then I think it should be used there.

But for the signing response, it always uses the shared data workspace. So I am not sure why we need both then.

If we can make it work with just the empty dir, that would be preferable. I'm ok with using the shared data workspace to make testing easier, but maybe the empty dir is not needed at all then and we don't need this parameter?

Reply:
I added it to the readme

Reply:
> Hmm, ok, so you have both an empty dir mounted to /shared and a shared data workspace. The config in the task is hardcoded to /shared/pubtools-sign-config.yaml:

Ah, that's correct. I started doing changes and then I realized why I added the parameter and started reverting the changes. I forgot to revert this one

> If we can make it work with just the empty dir, that would be preferable. I'm ok with using the shared data workspace to make testing easier, but maybe the empty dir is not needed at all then and we don't need this parameter?
Last time you wanted to have config stored outside of workspaces.data.path, that's why I move it to empty dir which is shared only within the task.

Reply:
> Last time you wanted to have config stored outside of workspaces.data.path, that's why I move it to empty dir which is shared only within the task.

I would have to see the context. In general we try to avoid using PVCs if possible. But if it's already there anyway, I don't see why not use it.